const intentionalError: number[] = [1, 2, 3, 5, 5, 6, 7, 8, 9, 10];

let error = intentionalError[23];
console.log(error);
